integration/raft/channel_participation_test.go

```diff
@@ -845,26 +974,25 @@ var _ = Describe("ChannelParticipation", func() {
 			})
 		})
 	})
-
 })
 
 // submit a transaction signed by the peer and ensure it was
 // committed to the ledger
-func submitPeerTxn(o *nwo.Orderer, peer *nwo.Peer, n *nwo.Network, orderers []*nwo.Orderer, expectedBlkNum int, expectedChannelInfo channelparticipation.ChannelInfo) {
+func submitPeerTxn(o *nwo.Orderer, peer *nwo.Peer, n *nwo.Network, expectedChannelInfo channelparticipation.ChannelInfo) {
 	env := CreateBroadcastEnvelope(n, peer, expectedChannelInfo.Name, []byte("hello"))
-	submitTxn(o, env, peer, n, orderers, expectedBlkNum, expectedChannelInfo)
+	submitTxn(o, env, peer, n, expectedChannelInfo)
 }
 
 // submit a transaction signed by the orderer and ensure it is
 // committed to the ledger
-func submitOrdererTxn(o *nwo.Orderer, peer *nwo.Peer, n *nwo.Network, orderers []*nwo.Orderer, expectedBlkNum int, expectedChannelInfo channelparticipation.ChannelInfo) {
+func submitOrdererTxn(o *nwo.Orderer, peer *nwo.Peer, n *nwo.Network, expectedChannelInfo channelparticipation.ChannelInfo) {
```

Comment:
Why is a peer involved in this at all?

Reply:
Oops, missed that. It's because this function originated as a helper to submit a transaction signed by a peer instead of an orderer.

Reply:
Done